modern/src/reports/ReportLayoutPage.js

```diff
@@ -1,42 +1,151 @@
 
 const useStyles = makeStyles(theme => ({
   root: {
-    height: '100%',
     display: 'flex',
-    flexDirection: 'column',
+    height: '100%',
+  },
+  drawerContainer: {
+    width: theme.dimensions.drawerWidth,
   },
+  drawer: {
+    width: theme.dimensions.drawerWidth,
+    [theme.breakpoints.down("md")]: {
+      width: theme.dimensions.tabletDrawerWidth,
+    }
+  }, 
   content: {
     flex: 1,
-    overflow: 'auto',
-    padding: theme.spacing(2),
+    padding: theme.spacing(5, 3, 3, 3),
   },
-  form: {
-    padding: theme.spacing(1, 2, 2),
+  drawerHeader: {
+    ...theme.mixins.toolbar,
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
+  },
+  backArrowIconContainer: {
+    '&:hover': {
+      backgroundColor:"transparent"
+    }  
+  },
+  toolbar: {
+    [theme.breakpoints.down("md")]: {
+      ...theme.mixins.toolbar,
+    }
   },
 }));
 
+const routes = [
+  { name: t('reportRoute'), href: '/reports/route', icon: <TimelineIcon /> },
+  { name: t('reportEvents'), href: '/reports/event', icon: <NotificationsActiveIcon /> },
+  { name: t('reportTrips'), href: '/reports/trip', icon: <PlayCircleFilledIcon /> },
+  { name: t('reportStops'), href: '/reports/stop', icon: <PauseCircleFilledIcon /> },
+  { name: t('reportSummary'), href: '/reports/summary', icon: <FormatListBulletedIcon /> },
+  { name: t('reportChart'), href: '/reports/chart', icon: <TrendingUpIcon /> },
+];
+
 const ReportLayoutPage = ({ children, filter }) => {
   const classes = useStyles();
+  const history = useHistory();
+  const theme = useTheme();
+  const [openDrawer, setOpenDrawer] = useState(false);
+  const location = useLocation();
+
+  const navigationList = (
+    <List disablePadding>
+      {routes.map((route, index) => (
+        <ListItem
+          disableRipple
+          component={Link}
+          key={`${route}${index}`}
+          button
+          to={route.href}
+          selected={route.href === location.pathname}>
+          <ListItemIcon>
+            {route.icon}
+          </ListItemIcon>
+          <ListItemText primary={route.name} />
+        </ListItem>
+      ))}
+    </List>
+  );
+
+  const drawerHeader = (
+    <> 
+      <div className={classes.drawerHeader}>
+        <IconButton 
+          className={classes.backArrowIconContainer} 
+          disableRipple>
+          <ArrowBackIcon />
+        </IconButton>
+        <Typography variant="h6" color="inherit" noWrap>
+          {t('reportTitle')}
+        </Typography> 
+      </div>
+      <Divider />
+    </>
+  );
+
+  const appBar = (
+    <AppBar position="fixed" color="inherit">
+      <Toolbar>
+        <IconButton
+          color="inherit"
+          aria-label="open drawer"
+          edge="start"
+          onClick={() => setOpenDrawer(!openDrawer)}
+          className={classes.menuButton}>
+          <MenuIcon />
+        </IconButton>
+        <Typography variant="h6" noWrap>
+          {t('reportTitle')}
```

Comment:
Can we show the title of the current page on mobile?

Reply:
Yes. I'm working on this.